ExpandableView/ExpandableView.cs

```diff
@@ -307,6 +311,7 @@ private void InvokeAnimation()
                         return;
                     }
                     RaiseStatusChanged(ExpandStatus.Expanded);
+                    SecondaryView.SizeChanged += OnSecondaryViewSizeChanged;
```

Comment:
don't think it's good idea to subscribe this event each time

Reply:
You have to. That's how you detect that the view's size has changed so that you can adjust the expander's height to accommodate.